components/brave_search/common/buildflags.gni

```diff
@@ -0,0 +1,7 @@
+import("//brave/build/config.gni")
+
+declare_args() {
+  enable_web_discovery_feature =
```

Comment:
I'd suggest to enable this fia Griffin

Reply:
That’s the plan for release

Reply:
We do need a buildflag here, but not with these conditions. Since this is an extension the buildflag should be `enable_web_discovery_feature = enable_extensions`

Reply:
yeah I mean why don't we enable it for nightly/beta via griffin?

Reply:
we discussed this over DM and decided not to have a feature flag for this since it's opt-in already.